doc/file-formats/README.md

```diff
@@ -0,0 +1,22 @@
+# File formats
+
+| Extension | Description
+| --------- | -----------
```

Comment:
Maybe link these to each relevant page?

Reply:
Requested changes in 094e0ce708c7decd8ffd17817fc08db1a56cf690.